misc/dist/linux/macros.godot

```diff
@@ -0,0 +1,41 @@
+#
+# RPM macros for godot
+#
+%__godot %{_bindir}/godot-headless
+
+# Macro: Prepare an export_presets.cfg file in the game's source.
+#        It will replace any of the custom_template/release fields
+#        with one that has a value of /usr/bin/godot-runner.
+# 
+# usage: %godot_prep [-f]
+#
+#   -f: Copy the installed export_presets.cfg file to game's source 
+#       even if one is already provided.
+#
+%godot_prep(f) \
+%__mkdir_p $HOME/.config/godot $HOME/.cache \
+if [ ! -f export_presets.cfg ] || [ -n "%{!-f:''}%{-f:"1"}" ]; then \
+  %__cp -fp %{_datadir}/godot/export_presets.cfg export_presets.cfg \
```

Comment:
This probably needs to be filled in by the Godot build system.

Reply:
That's not a file installed by Godot.